pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1242,7 +1242,11 @@ func GetPodAntiAffinityTerms(podAntiAffinity *v1.PodAntiAffinity) (terms []v1.Po
 	return terms
 }
 
-func getMatchingTopologyPairs(pod *v1.Pod, nodeInfoMap map[string]*schedulercache.NodeInfo) (*topologyPairsMaps, error) {
+// getTPMapMatchingExistingAntiAffinity calculates the following for each existing pod on each node:
+// (1) Whether it has PodAntiAffinity
+// (2) Whether any AffinityTerm matches the incoming pod
+// TODO(Huang-Wei): move this to metadta.go?
```

Comment:
considering that function is only invoked in `metadata.go` as an initilization helper, and similar function `getTPMapMatchingIncomingAffinityAntiAffinity()` is also defined/called in `metadata.go`. I think we should move it there as well.

Reply:
I'll move this function before squashing commits. As moving a function usually results in a tricky `diff` view.